modules/admin/admin-ui/src/main/resources/web/admin/apps/content-studio/js/app/browse/action/ContentTreeGridActions.ts

```diff
@@ -175,15 +146,17 @@ export class ContentTreeGridActions implements TreeGridActions<ContentSummaryAnd
         this.SHOW_NEW_CONTENT_DIALOG_ACTION.setEnabled(false);
         this.EDIT_CONTENT.setEnabled(this.anyEditable(contentSummaries));
         this.DELETE_CONTENT.setEnabled(this.anyDeletable(contentSummaries));
-        this.DUPLICATE_CONTENT.setEnabled(false);
+        this.DUPLICATE_CONTENT.setEnabled(contentSummaries.length == 1 ? true : false);
```

Comment:
Small refactoring: it's enough to do ```this.DUPLICATE_CONTENT.setEnabled(contentSummaries.length == 1);``` without the ```"? true : false"``` part.

Same with SORT_CONTENT.